Libraries/Components/DrawerAndroid/DrawerLayoutAndroid.android.js

```diff
@@ -159,8 +159,11 @@ var DrawerLayoutAndroid = React.createClass({
 
   render: function() {
     var drawStatusBar = Platform.Version >= 21 && this.props.statusBarBackgroundColor;
+    var drawerSubviewBackgroundColor = (this.props.style || {}).backgroundColor || 'white';
```

Comment:
what i meant was:
```
const styles = Stylesheet.create({
   drawerBackgroundColor: {
      backgroundColor: 'red',
   },
});
```

then passing `style={styles.drawerBackgroundColor}`. When you use `Stylesheet.create` `styles.drawerbackgroundColor` will be a number.